packages/node_modules/@webex/widget-space/src/container.js

```diff
@@ -176,8 +176,8 @@ export class SpaceWidget extends Component {
                   disabled={isMeetButtonDisabled}
                   onClick={() => props.handleActivityChange(at)}
                 >
-                  <Icon name={`icon-${at.buttonType}_16`} style={{paddingRight: call?.startTime ? '0.5rem' : 'inherit'}} />
-                  {call?.startTime && <Timer startTime={call?.startTime} />}
+                  <Icon name={`icon-${at.buttonType}_16`} style={{paddingRight: callStartTime ? '0.5rem' : 'inherit'}} />
+                  {callStartTime && <Timer startTime={callStartTime} />}
```

Comment:
Without this the timer gets started as soon as we click on join meeting, this makes sure that timer is only started when we actually join the meeting.